templates/breadcrumb.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Display breadcrumbs.
+ *
+ * @package WPAMPTheme
+ */
+
+if ( ! is_tax() ) {
+	return;
+}
+$term = get_term_by( 'slug', get_query_var( 'term' ), get_query_var( 'taxonomy' ) );
+?>
+<div id="crumb" class="py2 pl2 pr2 border-bottom travel-border-gray">
+	<span class="bold py1 pl1 pr1"><a class="travel-link" href="<?php echo esc_html( home_url() ); ?>"><?php esc_html_e( 'Home' ); ?></a></span>
+	<span class="travel-results-result-subtext">></span>
+	<span class="bold py1 pl1 pr1"><?php echo esc_html( $term->name ); ?></span>
```

Comment:
The breadcrumbs look really good. But I think the second item could be changed:

<img width="1281" alt="breadcrumb-taxonomy" src="https://user-images.githubusercontent.com/4063887/39204150-9ed90850-47bc-11e8-9c43-c24ebcdd317b.png">